COPYING

```diff
@@ -0,0 +1,21 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2012 - 2018 ENCRYPTED SUPPORT LP <adrelanos@riseup.net>
```

Comment:
Suggest adding a line: 2018 - Freedom of the Press Foundation <securedrop@freedom.press>